src/components/Settings/MaxSlippageSettings/index.tsx

```diff
@@ -37,15 +38,23 @@ const NUMBER_WITH_MAX_TWO_DECIMAL_PLACES = /^(?:\d*\.\d{0,2}|\d+)$/
 const MINIMUM_RECOMMENDED_SLIPPAGE = new Percent(5, 10_000)
 const MAXIMUM_RECOMMENDED_SLIPPAGE = new Percent(1, 100)
 
+function useFormatSlippageInput() {
```

Comment:
this is also completely valid to add as an optional parameter to the `formatSlippage` function

Reply:
v true. I think since it's limited to this file its fine for now